bindata/network/node-identity/managed/node-identity.yaml

```diff
@@ -29,6 +29,7 @@ spec:
       annotations:
         hypershift.openshift.io/release-image: {{.ReleaseImage}}
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        openshift.io/required-scc: hostnetwork-v2
```

Comment:
node-identity in managed is not a hostNetworked pod, this should work with restricted-v2?

Reply:
Thanks for this -- our monitor tests probably run only on self-hosted therefore I have missed this case. Fixed :+1: